### PR TITLE
Automatic commit: Move 'abap-environment-ips' from QA to Production

### PR DESCRIPTION
**Pull request by Liebherr**

### CODE REVIEW

File: tutorials/abap-environment-ips/abap-environment-ips.md [hunk 1]
```diff
@@ -0,0 +1,346 @@
...
+
+![Copy Technical User's ID](IAS_admin_copy_user_id.png)
+
+[DONE]
+[ACCORDION-END]
+
+[ACCORDION-BEGIN [Step 5: ](Configure Source in Identity Provisioning Service)]
+Identity provisioning requires to configure a so-called source system for user and user group data.
+
+Logon with your Identity Provisioning Manager user to your Identity Authentication tenant's identity provisioning UI (URL ends with path `/ips`, for example https://rapworkshop.accounts.ondemand.com/ips).
```
Comment (MichaelCzcz) [inline tutorials/abap-environment-ips/abap-environment-ips.md:127]: Log on

File: tutorials/abap-environment-ips/abap-environment-ips.md [hunk 2]
```diff
@@ -0,0 +1,346 @@
...
+![Get communication user credentials from service key](btp_service_key_get_credentials.png)
+
+[DONE]
+[ACCORDION-END]
+
+[ACCORDION-BEGIN [Step 6: ](Configure Target in Identity Provisioning Service )]
+Identity provisioning requires to configure a so-called target system for user and user group data.
+In this example, the target systems is an ABAP system in SAP BTP.
+
+Logon with your Identity Provisioning Manager user to your Identity Authentication tenant's identity provisioning UI (URL ends with path `/ips`, for example <https://rapworkshop.accounts.ondemand.com/ips>).
```
Comment (MichaelCzcz) [inline tutorials/abap-environment-ips/abap-environment-ips.md:217]: Log on

**Review by MichaelCzcz — CHANGES_REQUESTED** [review]
Please fix the two typos. There are many big images in each step with sometimes only one line of text between them. Please use a [numbered list](https://jam4.sapjam.com/wiki/show/MozDWaPczznL8NKeGEk0y9#mcetoc_1dtqfqchq22) for the instructions to improve readability.

**Comment by Liebherr** [comment]
Hi Michael, have done the changes and pushed again.
Thanks and Regards
Ulrike

**Pull request merged**